### DISCUSSION TITLE
Quickly unblock/block service for a certain client/IP/MAC

### DISCUSSION THREAD

**Discussion by mcrwanek** [AdguardTeam/AdGuardHome#2844] [Q&A]
I already shared this as a feature request but no feedback so far:
https://github.com/AdguardTeam/AdGuardHome/issues/2840

so in brief - is it somehow possible to quickly unblock/block service for a certain client_tag?
without going into "settings-client settings-client edit-block specific services-unclick/click this/that-save ?

thanks

**Comment by ameshkov** [comment]
Hi, answered you there!

**Reply by mcrwanek** [reply]
thanks!
